tests/openassetio/test/manager/test_main.py

```diff
@@ -98,8 +96,15 @@ def execute_cli(fixtures_path, *extra_args):
 
     @return `subprocess.CompletedProcess` The results of the invocation.
     """
-    all_args = ["python", "-m", "openassetio.test.manager"]
-    all_args.extend(["-f", fixtures_path])
+    # We may well be running inside a venv interpreter, but without
+    # the venv being active in the current environment. The effect
+    # of this is that the python on $PATH may not be the one that is
+    # correctly configured with the appropriate dependencies, so we
+    # attempt to use the same executable as the tests.
+    prog = sys.executable or "python"
```

Comment:
[consider] I'm not sure if the `or "python"` is a good idea.  Presumably this is only in case these tests are being executed in an embedded Python interpreter or something?  The test might work in that case, but likely it would fail and be harder to debug than allowing `None` to be passed to `subprocess` and fail earlier?

Reply:
Good call, removed [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/279/commits/bf78f09daa08c9a1a2d15eae9f0b1885c4a8ca75)